### ISSUE TITLE
visual layering issue with bar shutters

### ISSUE THREAD

**Issue by sharah-za-fluffy** [ScavStation/ScavStation#756]
#### Description of issue
the shutters are on top of the table and always visable and ugly even when open 

#### Difference between expected and actual behavior
it looks like ass
![Screenshot 2023-09-12 145509](https://github.com/ScavStation/ScavStation/assets/65055421/4dcbb43f-c0fd-4383-b502-fbb089920c36)

#### Steps to reproduce
go to bar

#### Specific information for locating

bar shutter

#### Length of time in which bug has been known to occur

12th september 2023

#### Client version, Server revision & Game ID

Client Version: 514
Server Revision: 161ef23615ddd2d78ae9f0794a5dacd7e331feeb - dev - 2023-09-10
Game ID: cpJ-dENg
Current map: Tradepost Mollusc

#### Issue bingo

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/ScavStation/ScavStation/issues)

**Comment by sharah-za-fluffy** [comment]
well that's strange because last version they where not visible on the
table unless closed

**Comment by Qumefox** [comment]
I agree with Sharah. Previously, shutters weren't visible unless actually shut.

**Comment by Qumefox** [comment]
If this isn't a bug, I might go through and change the 'open' sprite to just be fully transparent on scav, so the shutters won't be visible when open.  What few comments i've heard on the issue (me and sharah) prefer them to be invisible when open like they used to be.
